app/assets/stylesheets/spotlight/_exhibits_index.scss

```diff
@@ -0,0 +1,102 @@
+.exhibit-card {
+  box-sizing: border-box;
+  float: left;
+  height: $exhibit-card-height;
+  margin: 0 -1*$exhibit-card-gutter $exhibit-card-gutter $exhibit-card-gutter;
```

Comment:
`-1*$exhibit-card-gutter` should be written with a single space on each side of the operator: `-1 * $exhibit-card-gutter`